tests/rados/test_mon_addition_removal.py

```diff
@@ -0,0 +1,83 @@
+"""
+Module to test addition and removal of mon daemons
+
+"""
+import random
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.rados.monitor_workflows import MonitorWorkflows
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Module to test Reads balancer functionality on RHCS 7.0 and above clusters
```

Comment:
The description needs to be modified here to suite the module we are testing here.

Reply:
Done